programs/gdformat.nix

```diff
@@ -5,7 +5,7 @@ in
 {
   options.programs.gdformat = {
     enable = lib.mkEnableOption "gdformat";
-    package = lib.mkPackageOption pkgs "gdtoolkit" { };
+    package = lib.mkPackageOption pkgs "gdtoolkit_4" { };
```

Comment:
> gdtoolkit has been renamed to gdtoolkit_3 to distinguish from version 4

Unsure if we should support both gdtoolkit_3/4, maybe @haruki7049 can give some input on that.

Reply:
As long as it can be overwritten by the user, we don't need to support all of the options.

Reply:
I, too, don't think it is necessary to support all options if the user can override the options...